frontend/test/metabase/scenarios/question/new.cy.spec.js

```diff
@@ -86,6 +86,28 @@ describe("scenarios > question > new", () => {
   });
 
   describe("ask a (simple) question", () => {
+    it("should allow to search and select saved questions", () => {
+      cy.visit("/");
+      cy.contains("Ask a question").click();
+      cy.contains("Simple question").click();
+      cy.findByPlaceholderText("Search for a table...").type("Ord");
+      cy.contains("Orders, Count").click();
+      cy.contains("18,760");
+    });
+
+    it("should allow to search and select tables", () => {
+      cy.visit("/");
+      cy.contains("Ask a question").click();
+      cy.contains("Simple question").click();
+      cy.findByPlaceholderText("Search for a table...").type("Ord");
```

Comment:
This same pattern repeats 4 times which means that it can be extracted. You can even create a completely new `describe` blog dedicated only to search functionality for easier distinction.

Also, please avoid using `cy.contains()` whenever possible. `cy.findByText()` and `cy.findAllByText()` are much more strict and often times help us in detecting some regressions.

For example:
`cy.contains("Foo")` would work even if there are multiple strings "Foo" on the page. It also matches "Foobar" and "Foo Bar Baz". On top of that, it matches "BarFoo". You can imagine how that can lead to all sorts of problems if not used carefully.